packages/core/contracts/optimistic-asserter/implementation/OptimisticAsserter.sol

```diff
@@ -185,7 +183,6 @@ contract OptimisticAsserter is OptimisticAsserterInterface, Lockable, Ownable, M
         _oracleRequestPrice(assertionId, assertion.identifier, assertion.assertionTime);
 
         // Send dispute callback
-        // TODO: consider mergeing the isDisputeAlloowed into toe SSM callback (revert within callback to block).
```

Comment:
Not implementing yet as most of applications would not use disputer filters. If we were to do this then a dedicated dispute callback allowing to pass `msg.sender` as `disputeCaller` and `validateDisputers` property should be added to Escalation Manager interface.